src/web/lib/components/ThemeColorsEditor/index.js

```diff
@@ -36,8 +65,8 @@ class ThemeColorsEditor extends React.Component {
             return [
               <li
                 key={`dt-${idx}`}
-                className={classnames(name, "color", { selected: selectedColor === name })}
-                onClick={() => setSelectedColor({ name })}
+                className={classnames(name, "color", { selected: this.props.selectedColor && selectedColor === name })}
```

Comment:
I don't think you need the `this.props.selectedColor &&` here - `selectedColor === name` should also turn up false if `selectedColor` is `null`